pyproject.toml

```diff
@@ -30,6 +30,7 @@ certifi = "2023.7.22"
 sentry-sdk = "1.14.0"
 gunicorn = "22.0.0"
 whitenoise = "6.2.0"
+setuptools = "<75"
```

Comment:
suggestion: it would be good to have a todo comment to remind ourselves to remove this at some point

I expect we can probably resolve it now-ish by upgrading Poetry, but I support getting "use Poetry at all" into production first

Reply:
Good idea, I'll put a note in.

Reply:
Done.